config/application.rb

```diff
@@ -18,6 +18,18 @@ class Application < Rails::Application
     # Custom directories with classes and modules you want to be autoloadable.
     config.autoload_paths += [Rails.root.join('lib')]
 
+    config.before_initialize do
+      # Load up Errbit::Config with values from the environment
+      require Rails.root.join('config/load')
+
+      config.secret_key_base = Errbit::Config.secret_key_base
+      config.serve_static_assets = Errbit::Config.serve_static_assets
+    end
+
+    initializer "errbit.mongoid", before: "mongoid.load-config" do
```

Comment:
Now that you mention it. I suppose we might want to allow users to provide their own yaml file if they must. It would add some flexibility to allow it. Although I would like users to raise issues if they find something about errbit/mongoid that can't be configured through environment variables.

I have mixed thoughts on this, but my inclination is to err on the side of flexibility if it doesn't hurt anything, and allow the yaml file to override this config.